packages/bpk-stylesheets/webpack.config.babel.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.[jt]sx$/,
```

Comment:
Do we want to exclude js files from tests?

Reply:
This is not excluding test files, this `test` means its the file names to test to run through webpack and not has anything to do with tests :) 

This would also not be excluding it would be including `js`, `jsx`, `ts` and `tsx` files

Reply:
Ah right, bad phrasing - didn't mean actual testing 😬 
I was referring to the regex, I think it only matches jsx and tsx?

Reply:
Ooo you are right its missing the `?`after the `x` great spot!